Libraries/acdriver/Sources/Options.cpp

```diff
@@ -32,6 +32,8 @@ parseArgument(std::vector<std::string> const &args, std::vector<std::string>::co
         return libutil::Options::Current<bool>(&_printContents, arg);
     } else if (arg == "--compile") {
         return libutil::Options::Next<std::string>(&_compile, args, it);
+    } else if (arg == "--compile-output-filename") {
```

Comment:
Extension. I made it long (including compile output) because `--compile` includes both the source and the destination directory, and I wanted to be clear that this will affect the results of things (the `output`/destination) specifically when you use `compile`